src/bump-timestamp

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python3 -u
```

Comment:
We don't need this command anymore, right?

Reply:
We do, its the necessary part for `build` subcommand, as pruning part is being split to a separate script